### ISSUE TITLE
Fix automatic adding of magic images

### ISSUE THREAD

**Issue by Idrinth** [Idrinth/IDotD#163]
5PG seems to use even more broken image naming now - more images slip through.

**Comment by Ardenian** [comment]
One could, if the 5PG images cannot be loaded, externally load them from the dotd wikia, as second option. 
Alternatively, one could also think about storing them manually externally in some kind of data storage, since they are less likely to change whereas there might come new issues from 5PG in the future.

**Comment by Idrinth** [comment]
It's more of a naming issue, the api doesn't provide the correct names for the images :(

**Comment by Ardenian** [comment]
Something that will hapen with everything they change. They change something and over here something gets broken. Might be ways to prevent that, though externalizing the magic images might be one easy way, considerng they aren't file-size heavy either.

**Comment by Idrinth** [comment]
would need the correct name for those as well then - if I don't find them where I expected them...

**Comment by Ardenian** [comment]
Well, since you could name the images the way you want it would offer the advantage to be independent form their naming, though having to organize everything yourself.

**Comment by Idrinth** [comment]
I still need the source image to be found, ardenian, we are storing them locally once we found them, so they won't be requested from 5pg in any case.

**Comment by Theodotosp** [comment]
Do you need help on finding with this issue?

**Comment by Idrinth** [comment]
No, but thanks @Theodotosp , I just need to find the time to write the code to read the game's xml

**Comment by Ardenian** [comment]
Ahh, I see. Well, dotd wikia might be an addition to get the images.
